lib/api/apiUtils/object/locationConstraintCheck.js

```diff
@@ -31,11 +31,14 @@ export default function locationConstraintCheck(request, metaHeaders,
     const bucketLocationConstraint = bucket.getLocationConstraint();
     const requestEndpoint = request.parsedHost;
 
-    if (!BackendInfo.isValidControllingBackendParam(
+    const controllingBackend = BackendInfo.controllingBackendParam(
         objectLocationConstraint, bucketLocationConstraint,
-        requestEndpoint, log)) {
+        requestEndpoint, log);
+    if (!controllingBackend.isValid) {
         backendInfoObj = {
             err: errors.InvalidArgument,
+            description: 'Invalid location constraint for the' +
```

Comment:
why not just customize the description of the error itself and send that back and use it in the 3 api's and remove the customize description step from those api's since will already be done?